docs/guides/charts-overview.md

```diff
@@ -0,0 +1,86 @@
+# Sensitive Data Archive Helm Charts
+
+## Charts overview
+
+The `neicnordic` Helm repository contains the following charts (for configuration details click on the links below):
+
+- [sda-svc - SDA services](https://github.com/neicnordic/sensitive-data-archive/blob/main/charts/sda-svc/README.md)
+
+  This chart deploys the service components needed to operate the Sensitive Data Archive solution. The charts may include additional service components that might be beneficial for administrative operations or extending the Sensitive Data Archive solutions to facilitate different use cases.
```

Comment:
```suggestion
  This chart deploys the essential service components needed to operate the Sensitive Data Archive (SDA) solution. It may include additional service components that might be beneficial for administrative operations or extending the SDA solutions to facilitate different use cases.
```